### DISCUSSION TITLE
Customize or disable error stack trace.

### DISCUSSION THREAD

**Discussion by 1NDecent** [eclipse-birt/birt#1790] [Q&A] [answered]
An example call to http://localhost:8080/report/frameset?

Will produce this lengthy stack trace:
```
- There is no report design object available.
org.eclipse.birt.report.exception.ViewerException: There is no report design object available.
	at org.eclipse.birt.report.context.ViewerAttributeBean.__init(ViewerAttributeBean.java:254)
	at org.eclipse.birt.report.context.BaseAttributeBean.init(BaseAttributeBean.java:228)
	...
	at java.base/java.lang.Thread.run(Thread.java:1583)
```

How can I customize, or at least disable the stack trace?

I'm using 4.16 WebViewerExample, and tried fiddling with web.xml. ExceptionDialogFragment.jsp isn't related to the above case.

**Comment by speckyspooky** [comment, marked as answer]
If you talk about the error-dialog on the viewer then you will the the short error-message on the dialog
and if you shows the details then you will see the whole stack trace.
For the details is currently no special configuration available to reduce him.

But may be you talk about another place instead of the error-dialog.
